### PR TITLE
Rails 5.2 upgrade

### PR DESCRIPTION
**Pull request by Souravirus**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [X] code is in uniquely-named feature branch and has no merge conflicts
* [X] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
It keeps on coming! 😄

**Comment by Souravirus** [comment]
Yeah :smile:

**Comment by jywarren** [comment]
Oh, wow, passing already? Is this ready?

**Comment by Souravirus** [comment]
No, some deprecation warning to be cleared then it will be ready

**Comment by jywarren** [comment]
Cool, how about now? and... WHATS AFTER 5.2??? 😆

**Comment by Souravirus** [comment]
Yeah only one deprecation warning is left. I am behind it and after that we will be ready to merge. Actually, we should stop at rails 5.2. what do you think? I guess we should do rails 6.0 update later some time. And I don't know what to do after rails 5.2 upgrade. Is there any project which I could work. I would be really happy working at it. Thank you!!

**Comment by Souravirus** [comment]
I guess its ready to merge @jywarren. That deprecation warning was due to sqlite3 gem which will removed by them later.

File: app/controllers/notes_controller.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def places
       .references(:term_data)
       .where('term_data.name = ?', 'chapter')
       .group('node.nid')
-      .order('max(node_revisions.timestamp) DESC, node.nid')
+      .order(Arel.sql('max(node_revisions.timestamp) DESC, node.nid'))
```
Comment (jywarren) [inline app/controllers/notes_controller.rb]: Whoa, what's this? How does it work?

Comment (Souravirus) [reply app/controllers/notes_controller.rb]: It's basically removing the deprecation warning : 
`DEPRECATION WARNING: Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "max(node_revisions.timestamp) DESC, node.nid". Non-attribute arguments will be disallowed in Rails 6.0. This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passed by wrapping them in Arel.sql(). (called from places at /app/app/controllers/notes_controller.rb:25)`

and as they have written in the warning itself that `Known-safe values can be passed by wrapping them in Arel.sql()`. So, I followed them.

**Comment by jywarren** [comment]
OK, awesome - maybe we should insert a comment by this usage referencing
this OK usage?

**Comment by Souravirus** [comment]
Ok about this arel usage right? I will insert a comment. Thank you!!

**Comment by Souravirus** [comment]
@jywarren, Check the comments I have added. Thank you!!

**Comment by jywarren** [comment]
Looks great! Ill merge now. 🎉

**Comment by Souravirus** [comment]
Thanks @jywarren

**Pull request merged**